tests/e2e/e2e_http_ingress_test.go

```diff
@@ -54,6 +55,27 @@ var _ = OSMDescribe("HTTP ingress",
 			Expect(Td.WaitForPodsRunningReady(destNs, 60*time.Second, 1)).To(Succeed())
 
 			// Install nginx ingress controller
```

Comment:
I suggest moving the installation of nginx ingress into a separate helper function so that new ingress tests can reuse the code.

Reply:
Are there other tests that need these same changes right now? That should be easy enough to do once we add another ingress test.

Reply:
Not right now, but I am certain ingress deployment on Kind vs non-Kind clusters needs to be made reusable across tests.